content/writing/newsletter.md

```diff
@@ -0,0 +1,14 @@
+# Newsletter
+Here are some guidelines for writing the Developer Relations Newsletter. It's a work in progress.
+
+## Subject
+It is [recommended by Mailchimp](https://mailchimp.com/help/best-practices-for-email-subject-lines/) to use no more than 9 words and 60 characters since multiple people will read your email on a mobile device.
+
+## Headers
+We use proper English capitalization rules and try to keep them short.
```

Comment:
We could also add this
> Capitalize the first word except for conjunction words like but, and, or etc.

Reply:
It is what "English capitalization rules" meant, but I'll change for your wording as it's clearer.